app/models/user/roles.rb

```diff
@@ -1,9 +1,10 @@
 module User::Roles
   def founder? = roles.include?(:founder)
   def admin? = roles.include?(:admin)
-  def staff? = roles.include?(:staff)
-  def maintainer? = roles.include?(:maintainer)
-  def supermentor? = roles.include?(:supermentor)
+  def staff? = roles.include?(:staff) || admin?
+  def maintainer? = roles.include?(:maintainer) || admin?
```

Comment:
And staff here too?

Reply:
I was unsure of that at first, but it's probably fine. Updated